lib/output.js

```diff
@@ -46,7 +46,6 @@ function processOutputs(outputs) {
     else {
       throw "Unexpected output format: " + output.toString();
     }
-    source = removeSourceMaps(source || '');
```

Comment:
The reasons is that babel already includes the sourcemaps into the output.source. output.sourceMap is emtpy, hence this just removes the already existing sourcemap.